src/editor/index.js

```diff
@@ -286,7 +279,6 @@ Inspector.prototype = {
    */
   open: function (focusEl) {
     this.opened = true;
-    Events.emit('inspectortoggle', true);
```

Comment:
The open() function may be called from the toggle function here when using the shortcut., so you need to set the global store isInspectorEnabled. This is also related to my comment https://github.com/3DStreet/3dstreet/pull/914/files#r1838538120

Reply:
removed the shortcut since i dont think we really need to support that at the moment